DIApp/src/main/java/Build/Computer.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * 
```

Comment:
usually this comment, the class comment, should be right above the public class line